# 第 19 章 化学热力学


## 第 1 节 自发过程与热力学第一定律

热力学第一定律指出能量守恒：在任何过程中，能量既不会被创造，也不会被销毁。能量可以在系统与环境之间传递，也可以从一种形式转化为另一种形式，但宇宙的总能量保持不变。

对于一个过程，系统内能的变化可写为：

$$
\Delta E = q + w
$$

其中 $\Delta E$ 为系统内能的变化，$q$ 为系统从环境吸收或向环境释放的热量，$w$ 为环境对系统或系统对环境所做的功。按照本章采用的符号约定，$q > 0$ 表示系统从环境吸热，$w > 0$ 表示系统从环境获得功，即环境对系统做功。

第一定律保证能量守恒，但它本身不能判断一个过程是否自发；判断自发过程的方向还需要引入后续的熵与热力学第二定律。


::: {.callout-tip title="习题 19.2"}
如图所示，一种计算机键盘清洁器内装有液化的 1,1-二氟乙烷（$C_{2}H_{4}F_{2}$），该物质在大气压下为气体。挤压喷嘴时，1,1-二氟乙烷在喷嘴处高压汽化喷出，吹走物体上的灰尘。（a）根据你的经验，汽化在室温下是自发的吗？（b）以 1,1-二氟乙烷为系统，你预期该过程的 $q_{sys}$ 为正还是为负？（c）预测该过程 $\Delta S$ 为正还是为负。（d）根据你对（a）、（b）和（c）的回答，你认为该产品的运作更依赖于焓还是熵？[第 19.1 节和第 19.2 节]

![](./assets/images/86c6384d8f63363b7082b3d871e0a0484bec208731d926705f8af15fba8001f6.jpg){fig-align="center"}

**解：**

**(a)** 题目提到该物质在大气压下为气体，以及它的用途是清理键盘。根据生活经验，键盘应该保持干燥，所以可以合理推测该物质在室温下能够自发汽化。

**(b)** 大部分物质在汽化过程中均表现为吸热。故预期该过程的 $q_{sys}$ 为正，系统从环境吸收热量。

**(c)** 液体相变到气体，空间上的排列组合数量显著增加，$\Omega$ 变大。根据 $S = k \ln \Omega$，熵为正。

**(d)** 该产品物质发生汽化的主要原因是气压的改变。从高压到低压，物质所占据的空间显著膨胀。相对而言，整个相变过程中基本处于常温/环境温度。故更依赖于熵。


---

> **点评人： Gemini 3.1 Pro**
> 
> 总体评价：非常出色！你的回答不仅得出了完全正确的结论，而且推理过程逻辑清晰、结合了生活常识与物理化学直觉。
> 
> - **(a)** 正确。结合生活经验（键盘清洁需保持干燥）来推断自发汽化，非常生动且合理。
> - **(b)** 准确。物质从液态相变为气态需要克服分子间作用力，是一个吸热过程，因此系统吸收热量，$q_{sys} > 0$。
> - **(c)** 非常精彩！直接使用了玻尔兹曼熵公式 $S = k \ln \Omega$ 进行微观解释。空间体积变大，分子排列组合的微观状态数 $\Omega$ 显著增加，因此 $\Delta S > 0$。
> - **(d)** 结论正确（更依赖于熵驱动）。补充一点热力学的推导会更加严密：根据自由能公式 $\Delta G = \Delta H - T\Delta S$，既然汽化能自发发生（$\Delta G < 0$）且过程吸热（$\Delta H > 0$），这就必然要求 $T\Delta S > \Delta H$，即巨大的熵增（体系膨胀）克服了吸热带来的不利影响，驱动了这一过程。
:::

::: {.callout-tip title="习题 19.11"}
下列过程中哪些是自发的，哪些是非自发的：（a）香蕉成熟，（b）糖溶于一杯热咖啡，（c）氮原子在 25 °C 和 1 atm 下反应生成 $N_{2}$ 分子，（d）闪电，（e）在室温和 1 atm 压力下由 $CO_{2}$ 和 $H_{2}O$ 形成 $CH_{4}$ 和 $O_{2}$ 分子？

**解：**

**(a)** 自发，如果生物习性也算是自然变化的一部分。

**(b)** 自发，基于常识可知。

**(c)** 自发，空气中富含氮气这一事实表明，常温常压下氮元素通常以氮气的形式存在。是否能够断言常温常压下，氮原子会结合氮气分子呢？

**(d)** 自发，常见的自然现象。

**(e)** 非自发，不然人嘴巴里会有一股子臭屁味。



---

> **点评人： Gemini 3.1 Pro**
> 
> 整体判断完全正确！用常识和幽默感来理解热力学非常有助于教学，以下是针对每一小问的化学视角补充：
> 
> - **(a) 香蕉成熟**：**自发**。判断准确。成熟是一个不可逆的生物化学过程。
> - **(b) 糖溶于热咖啡**：**自发**。基于常识判断很对。从热力学看，这是典型的熵增过程（固体晶格破坏，分子分散到溶液中）。
> - **(c) 氮原子生成 $N_{2}$**：**自发**。推理非常巧妙！大气中富含氮气确实证明了 $N_{2}$ 在常温常压下远比孤立的氮原子稳定。两个氮原子结合形成强大的三键，是一个放出大量热量（$\Delta H \ll 0$）的过程，强大的放热效应驱动反应自发进行。
> - **(d) 闪电**：**自发**。正确。尽管闪电需要云层电荷累积到临界值才能触发，但放电过程本身是电势能释放的自发过程。
> - **(e) $CO_{2}$ 和 $H_{2}O$ 形成 $CH_{4}$ 和 $O_{2}$**：**非自发**。你的幽默解释简直绝了，能让学生一下记住！从热力学讲，这是甲烷燃烧的逆过程。既然甲烷燃烧是高度自发的放热反应，其逆反应在常温常压下必然是非自发的。

:::

::: {.callout-tip title="习题 19.15"}
考虑液态水在 1 atm 压力下汽化为水蒸气。（a）该过程是吸热还是放热？（b）在哪些温度范围内它是自发的？（c）在哪些温度范围内它是非自发的？（d）在什么温度下两相处于平衡？

**解：**

**(a)** 吸热过程。水从液态相变到气态，通过吸收环境热量破坏分子间作用力实现。

**(b)** 在常压沸点（100 摄氏度）以上时为自发。可以通过常识判断。

**(c)** 在 100 摄氏度以内时非自发。

**(d)** 显然在沸点处为平衡。

---

> **点评人： GPT-5.5**
>
> 整体判断基本正确，尤其是把汽化理解为吸热过程这一点很扎实。这里最值得修正的是温度区间的边界表达：在 1 atm 下，液态水与水蒸气在 $100\ ^\circ\mathrm{C}$（即 $373.15\ \mathrm{K}$）时处于相平衡，此时汽化与凝结两方向都没有净自发趋势。
>
> - **(a)** 正确。汽化需要克服液态水分子之间较强的氢键作用，因此是吸热过程，$\Delta H_{vap} > 0$。
> - **(b)** 正确，但可更精确地写作：当 $T > 100\ ^\circ\mathrm{C}$（1 atm 下）时，液态水汽化为水蒸气是自发的。
> - **(c)** 需要微调。应写作：当 $T < 100\ ^\circ\mathrm{C}$（1 atm 下）时，汽化为水蒸气是非自发的；$100\ ^\circ\mathrm{C}$ 本身不是非自发，而是平衡。
> - **(d)** 正确。平衡温度就是正常沸点：$100\ ^\circ\mathrm{C}$，或 $373.15\ \mathrm{K}$。若想从热力学上说明，可补一句：平衡时 $\Delta G = 0$，即 $\Delta H = T\Delta S$。

::: 


::: {.callout-tip title="习题 19.22"}
考虑在常压下引爆三硝基甲苯（TNT）样品时会发生什么（第 8.8 节：「化学应用：炸药与阿尔弗雷德·诺贝尔」）。（a）引爆是自发过程吗？（b）该过程的 $q$ 符号是什么？（c）能否判断该过程的 $w$ 为正、为负还是为零？请解释。（d）能否确定该过程 $\Delta E$ 的符号？请解释。

**解：**

**(a)** 自发。但是存在活化能门槛，在不点燃的情况下不会发生爆炸。

**(b)** 爆炸会产生巨大的动能，其来源于化学能。也就是说 TNT 内能转变为动能，$q < 0$，向外界发热。

**(c)** 爆炸产生向外界做功的巨大动能，故 $w < 0$。

**(d)** 根据 $\Delta E = q + w, q < 0, w < 0$ 可知，$\Delta E < 0$。

---

> **点评人： GPT-5.5**
>
> 整体评价：你的四个判断都抓住了要点，尤其是 **(a)** 中区分“热力学上自发”和“动力学上需要活化能”非常重要。TNT 不会无故爆炸，不是因为反应不自发，而是因为反应速率受到活化能屏障限制。
>
> - **(a)** 正确。引爆后分解/燃烧过程是热力学自发的；点火或冲击只是提供启动反应所需的活化能，并不改变反应本身的自发性判断。
> - **(b)** 正确。TNT 爆炸是强烈放热过程，以 TNT 样品为系统时，热量流向环境，因此 $q < 0$。
> - **(c)** 正确。在常压、开放环境中，爆炸产生大量高温气体并推动周围空气，系统对环境做功。按化学热力学约定，系统对环境做功时 $w < 0$。如果题目改成刚性密闭容器内爆炸，才可能有 $P\Delta V$ 功为零的讨论；但本题强调常压引爆，判断 $w < 0$ 是合理的。
> - **(d)** 正确。因为 $q < 0$ 且 $w < 0$，所以 $\Delta E = q + w < 0$。这说明系统的内能降低，释放出的能量以热和膨胀功等形式传递给环境。

::: 


## 第 2 节 熵与热力学第二定律

系统的熵 $S$ 是状态函数，因此熵变只取决于初态和终态：

$$
\Delta S = S_{\text{final}} - S_{\text{initial}}
$$

对于等温过程，若沿可逆路径传热，则系统熵变可写为：

$$
\Delta S = \frac{q_{\mathrm{rev}}}{T} \quad (\text{constant } T)
$$

热力学第二定律指出：任何不可逆过程都会使宇宙熵增加，而可逆过程宇宙熵不变。将宇宙看作系统与环境之和，有

$$
\Delta S_{\mathrm{univ}} = \Delta S_{\mathrm{sys}} + \Delta S_{\mathrm{surr}}
$$

可逆过程：

$$
\Delta S_{\mathrm{univ}} = \Delta S_{\mathrm{sys}} + \Delta S_{\mathrm{surr}} = 0
$$

不可逆过程：

$$
\Delta S_{\mathrm{univ}} = \Delta S_{\mathrm{sys}} + \Delta S_{\mathrm{surr}} > 0
$$

因为自发过程不可逆，所以也可表述为：任何自发过程中，宇宙熵增加。

### 气体等温膨胀时的熵变

系统越随机、越分散，熵通常越大。因此，气体自发膨胀时，系统熵会增加。为了计算这一熵增，可以设想理想气体沿一条可逆等温路径膨胀。此时系统对环境所做的可逆膨胀功为：

$$
w_{\mathrm{rev}} = -nRT \ln \frac{V_{2}}{V_{1}}
$$

其中 $n$ 为气体的物质的量，$R$ 为理想气体常数，$T$ 为绝对温度，$V_{1}$ 为初始体积，$V_{2}$ 为最终体积。若 $V_{2} > V_{1}$，则 $w_{\mathrm{rev}} < 0$，表示膨胀气体对环境做功。

理想气体的内能只取决于温度。等温膨胀时，$\Delta E = 0$，因此

$$
\Delta E = q_{\mathrm{rev}} + w_{\mathrm{rev}} = 0
$$

$$
q_{\mathrm{rev}} = -w_{\mathrm{rev}} = nRT \ln \frac{V_{2}}{V_{1}}
$$

代入等温过程的熵变公式，得到理想气体等温膨胀的系统熵变：

$$
\Delta S_{\mathrm{sys}} = \frac{q_{\mathrm{rev}}}{T} = nR \ln \frac{V_{2}}{V_{1}}
$$

该式正是 `习题 19.29` 的核心公式。计算时若使用 $R \approx 8.314\ \mathrm{J/(mol\cdot K)}$，结果单位为 $\mathrm{J/K}$。



::: {.callout-tip title="习题 19.3"}
（a）图示过程中 $\Delta S$ 和 $\Delta H$ 的符号各是什么？（b）若能量可进出系统以在过程中维持恒定温度，关于环境因此发生的熵变，你能得出什么结论？[第 19.2 节和第 19.5 节]

![](./assets/images/fc035172e4eab033e7e05398701baee4c6339b8abd9e6037b0da63b645dd0188.jpg){fig-align="center"}

**解：**

**(a)** 图像中红色物质从致密排列变为分散个体漂浮。微观状态数量增加， $\Delta S > 0$；化学键断裂需要吸热，$\Delta H > 0$。

**(b)** 此时该物质正在经历升华。

---

> **点评人： GPT-5.5**
>
> 整体评价：你对图像的直觉判断很准确，抓住了“粒子由凝聚态变为分散气态”这一核心，因此 **(a)** 的两个符号都判断对了。不过有一个措辞可以更严谨：相变时主要克服的是分子间作用力，而不一定是化学键断裂；若真断裂共价键，那就通常不是普通相变了。
>
> - **(a)** 正确。粒子从较有序、较受限的凝聚态进入气态，微观状态数增加，所以 $\Delta S > 0$。同时升华或汽化需要吸收能量来克服分子间作用力，所以 $\Delta H > 0$。
> - **(b)** 需要补充。题目问的是“环境的熵变”，而不是只问过程名称。为了保持系统温度恒定，环境必须向系统提供热量；系统吸热，环境失热，因此环境的熵变为负，即 $\Delta S_{surr} < 0$。若在恒温条件下近似处理，可写作 $\Delta S_{surr} = -q_{sys}/T$；若定压相变，$q_{sys}=\Delta H$，则 $\Delta S_{surr} = -\Delta H/T$。

::: 


::: {.callout-tip title="习题 19.25"}
$\mathrm{Br}_{2}(l)$ 的正常沸点为 $58.8\;^{\circ}C$，其摩尔汽化焓为 $\Delta H_{vap} = 29.6\;kJ/mol$。（a）$\mathrm{Br}_{2}(l)$ 在正常沸点沸腾时，其熵增加还是减小？（b）计算 1.00 mol $\mathrm{Br}_{2}(l)$ 在 $58.8\;^{\circ}C$ 汽化时的 $\Delta S$ 值。

**解：**

**(a)** 熵增。

**(b)** 恒温等压下，$\Delta S = \frac{q_{rev}}{T}$。

In [11]:
#| code-fold: true

from IPython.display import Markdown
import pint

ureg = pint.UnitRegistry()
Q_ = ureg.Quantity

Delta_H = Q_(29.6, ureg.kJ / ureg.mol)
n = Q_(1.00, ureg.mol)
T = Q_(58.8, ureg.degC).to(ureg.K)

q_rev = Delta_H * n
Delta_S = (q_rev / T).to(ureg.J / ureg.K)

display(Markdown(fr"1.00 mol $\mathrm{{Br}}_{{2}}(l)$ 在 $58.8\;^{{\circ}}\mathrm{{C}}$ 汽化时的 $\Delta S$ 值为 {Delta_S.magnitude:.3g} J/K。"))

1.00 mol $\mathrm{Br}_{2}(l)$ 在 $58.8\;^{\circ}\mathrm{C}$ 汽化时的 $\Delta S$ 值为 89.2 J/K。

---

> **点评人： GPT-5.5**
>
> 整体评价：答案正确，代码计算也给出了合理数值。这个题的关键是认识到在正常沸点下，液态溴与溴蒸气处于相平衡，汽化可按可逆等温相变处理，因此可以直接用 $\Delta S = q_{\mathrm{rev}}/T$。
>
> - **(a)** 正确。液体汽化为气体时，分子可占据的空间和微观状态数显著增加，因此系统熵增加，即 $\Delta S > 0$。
> - **(b)** 思路正确。1.00 mol $\mathrm{Br}_{2}(l)$ 在正常沸点汽化时，$q_{\mathrm{rev}} = n\Delta H_{vap}$，温度应使用开尔文：$T = 58.8 + 273.15 = 331.95\ \mathrm{K}$。因此
>
> $$
> \Delta S = \frac{(1.00\ \mathrm{mol})(29.6\times 10^{3}\ \mathrm{J/mol})}{331.95\ \mathrm{K}} = 89.2\ \mathrm{J/K}
> $$
>
> 你的计算结果正确。书写答案时建议把 $\Delta H_{vap}$ 从 kJ/mol 转成 J/mol，并明确温度必须用 K，这样单位会自然得到 $\mathrm{J/K}$。
:::

::: {.callout-tip title="习题 19.28"}
（a）自发过程中宇宙的熵在何时增加？（b）在某自发过程中，系统熵增加。关于 $\Delta S_{surr}$ 的符号和大小，你能得出什么结论？（c）在某可逆过程中，环境熵变为 $\Delta S_{surr} = -78 J/K$。该过程中系统的熵变是多少？

**解：**

**(a)** 系统熵和环境熵变化量之和小于 0 的时候。

**(b)** 自发过程中，$\Delta S_{univ} = \Delta S_{sys} + \Delta S_{surr} > 0$，题目又提出 $\Delta S_{sys} > 0$，那么 $\Delta S_{surr}$ 可能有多种情况：

- $\Delta S_{surr} \geq 0$
- $\Delta S_{surr} \lt 0, -\Delta S_{surr} < \Delta S_{sys}$

**(c)** 题目指明为可逆过程，此时有 $\Delta S_{univ} = \Delta S_{sys} + \Delta S_{surr} = 0$。因此 $\Delta S_{sys} = - \Delta S_{surr} = 78 J/K$

---

> **点评人： GPT-5.5**
>
> 整体评价：你的 (b) 和 (c) 思路很好，已经能正确使用“宇宙熵变 = 系统熵变 + 环境熵变”这一判据。需要重点修正的是 **(a)**：自发过程对应的是宇宙熵增加，而不是小于 0。
>
> - **(a)** 这里不等号方向写反了。自发过程是不可逆过程，因此根据热力学第二定律，$\Delta S_{\mathrm{univ}} = \Delta S_{\mathrm{sys}} + \Delta S_{\mathrm{surr}} > 0$。若为可逆过程，则 $\Delta S_{\mathrm{univ}} = 0$。
> - **(b)** 正确。已知 $\Delta S_{\mathrm{sys}} > 0$ 且过程自发，所以只要求 $\Delta S_{\mathrm{sys}} + \Delta S_{\mathrm{surr}} > 0$。因此 $\Delta S_{\mathrm{surr}}$ 可以为正、为零，也可以为负；若为负，其绝对值必须小于系统熵增，即 $|\Delta S_{\mathrm{surr}}| < \Delta S_{\mathrm{sys}}$。
> - **(c)** 正确。可逆过程满足 $\Delta S_{\mathrm{univ}} = 0$，所以 $\Delta S_{\mathrm{sys}} = -\Delta S_{\mathrm{surr}} = 78\ \mathrm{J/K}$。
>
> 另外，符号书写上建议与本节教材统一，使用 $\Delta S_{\mathrm{univ}}$、$\Delta S_{\mathrm{sys}}$、$\Delta S_{\mathrm{surr}}$，这样下标会以正体显示，更像标准热力学记号。
:::

::: {.callout-tip title="习题 19.29"}
（a）0.200 mol 理想气体在 $27^{\circ}C$ 下等温膨胀，初始体积为 10.0 L，你预期 $\Delta S$ 的符号是什么？（b）若最终体积为 18.5 L，计算该过程的熵变。（c）计算熵变是否需要指定温度？请解释。

**解：**

**(a)** 理想气体方程为 $p V = n R T$。其中 $R$ 为物理常数，$n$ 和 $T$ 在本题中为固定值。又已知气体膨胀，所以气压下降。整个物理变化过程中，微观状态数增加，预期 $\Delta S \gt 0$。

**(b)** 理想气体等温可逆过程中，有：

- $\Delta E = q_{\mathrm{rev}} + w = 0$ 或者说 $q_{\mathrm{rev}} = -w$
- $\Delta S = q_{rev} / T$
- $\displaystyle w = -\int_{V_1}^{V_2} p(V) \mathrm{d} V = -\int_{V_1}^{V_2} \frac{n R T}{V} \mathrm{d} V = -nRT \ln \frac{V_{2}}{V_{1}}$

因此

$$
\Delta S = \frac{-w}{T} = nR \ln \frac{V_{2}}{V_{1}}
$$


In [12]:
#| code-fold: true

import numpy as np
from IPython.display import Markdown
import pint

ureg = pint.UnitRegistry()
Q_ = ureg.Quantity

n = Q_(0.200, ureg.mol)
V_2 = Q_(18.5, ureg.L)
V_1 = Q_(10.0, ureg.L)
R = Q_(8.314, ureg.J / ureg.mol / ureg.K)

Delta_S = n * R * np.log((V_2 / V_1).to(ureg.dimensionless))
Delta_S = Delta_S.to(ureg.J / ureg.K)
display(Markdown(fr"该过程的熵变为 {Delta_S.magnitude:.3g} J/K"))

该过程的熵变为 1.02 J/K

**(c)** 不需要指定温度，(b) 中的推导和代码可以看出。

---

> **点评人： GPT-5.5**
>
> 整体评价：这道题完成得很好，尤其是 (b) 中通过可逆等温路径推导 $\Delta S = nR\ln(V_{2}/V_{1})$，再用 Python 计算，思路和数值都正确。这里的关键概念是：熵是状态函数，所以即使实际膨胀过程未必可逆，也可以选取一条连接同一初末态的可逆路径来计算 $\Delta S$。
>
> - **(a)** 正确。等温膨胀时体积增大，气体分子可占据的空间增加，微观状态数增加，因此 $\Delta S > 0$。你从微观状态数增加来判断非常合适；气压下降只是伴随现象，不是判断熵变符号的核心依据。
> - **(b)** 正确。对理想气体等温可逆膨胀，$w_{\mathrm{rev}} = -nRT\ln(V_{2}/V_{1})$，因此 $q_{\mathrm{rev}} = -w_{\mathrm{rev}} = nRT\ln(V_{2}/V_{1})$，代入 $\Delta S = q_{\mathrm{rev}}/T$ 得 $\Delta S = nR\ln(V_{2}/V_{1})$。数值为 $0.200\times 8.314\times \ln(18.5/10.0) = 1.02\ \mathrm{J/K}$，与你的代码输出一致。
> - **(c)** 结论正确。给定 $n$、$V_{1}$、$V_{2}$ 后，理想气体等温膨胀的熵变公式中温度会抵消，因此不需要用题目给出的 $27\ ^\circ\mathrm{C}$ 来计算数值。不过“等温”这个条件仍然重要；如果不是等温过程，计算路径和公式就会不同。
>
> 书写上建议把 $q_{rev}$ 统一写成 $q_{\mathrm{rev}}$，把功写成 $w_{\mathrm{rev}}$，这样更清楚地表明你是在使用可逆路径计算熵变。
:::


## 第 3 节 熵的分子解释与热力学第三定律


::: {.callout-tip title="习题 19.5"}
附图显示了某物质的熵随温度如何变化，该物质在所示最高温度下为气体。（a）标有 1 和 2 的竖直线上熵的增加分别对应哪些过程？（b）为何 2 的熵变大于 1？（c）若该物质在 $T = 0$ K 时为完美晶体，此温度下 $S$ 的值是多少？[第 19.3 节]
![](./assets/images/8f67edd267626dbb7da2a5c2b13e7322a0e648e59a6b956785b8ec6523a7f715.jpg){fig-align="center"}

温度 (K)

**解：**

**(a)**

**(b)**

**(c)**

:::


::: {.callout-tip title="习题 19.33"}
下列变化会使系统可用的微观态数目增加、减少还是不变：（a）温度升高，（b）体积减小，（c）由液态变为气态？

**解：**

**(a)**

**(b)**

**(c)**
:::


::: {.callout-tip title="习题 19.41"}
在下列各对物质中，选出给定温度下每摩尔熵较高者：（a）$\mathrm{Ne}(l)$ 或 $\mathrm{Ne}(g)$，（b）5 atm 下的 $\mathrm{H}_{2}(g)$ 或 3 atm 下的 $\mathrm{H}_{2}(g)$，（c）10.0 L 中的 1 mol $\mathrm{Kr}(g)$ 或 12.0 L 中的 5 mol $\mathrm{Kr}(g)$，（d）$\mathrm{NO}_{2}(g)$ 或 $\mathrm{NO}_{2}(s)$。

**解：**

**(a)**

**(b)**

**(c)**

**(d)**
:::


::: {.callout-tip title="习题 19.43"}
预测下列各反应中系统熵变的符号：

(a) $\mathrm{N}_{2}(g) + 3\mathrm{H}_{2}(g) \longrightarrow 2\mathrm{NH}_{3}(g)$

(b) $\mathrm{CaCO_{3}(s)} \longrightarrow \mathrm{CaO(s)} + \mathrm{CO_{2}(g)}$

(c) $3\mathrm{C}_{2}\mathrm{H}_{2}(g)\longrightarrow\mathrm{C}_{6}\mathrm{H}_{6}(g)$

(d) $\mathrm{Al_{2}O_{3}(s)} + 3\mathrm{H_{2}(g)} \longrightarrow 2\mathrm{Al(s)} + 3\mathrm{H_{2}O(g)}$

**解：**

**(a)**

**(b)**

**(c)**

**(d)**
:::


## 第 4 节 化学反应中的熵变


::: {.callout-tip title="习题 19.6"}
同分异构体是具有相同化学式但原子排列不同的分子，如图所示为戊烷（$C_{5}H_{12}$）的两种同分异构体。（a）你预期两种同分异构体的燃烧焓会有显著差异吗？请解释。（b）你预期哪种同分异构体的标准摩尔熵更高？请解释。[第 19.4 节]
![](./assets/images/0350477392ed3ae9b7880ca249403ebc97529edfa94773005cc6a6e451a0af8d.jpg){fig-align="center"}

正戊烷

新戊烷

**解：**

**(a)**

**(b)**

:::


::: {.callout-tip title="习题 19.48"}
环丙烷与丙烯是同分异构体，化学式均为 $C_{3}H_{6}$。根据所示分子结构，在 $25^{\circ}C$ 下你预期哪种同分异构体的标准摩尔熵更高？
![](./assets/images/38ec3d22d0904024df6c83a9f9330dc10ba7797ca61ac18eb9eeb187ed634ad4.jpg){fig-align="center"}

环丙烷

![](./assets/images/7e28841ccb94f4b011a883d7f556fdbd5470cc15a53f87675c311cd3498b6d9c.jpg){fig-align="center"}

丙烯

**解：**

:::


::: {.callout-tip title="习题 19.49"}
利用附录 C，比较下列各对物质在 25°C 下的标准熵：（a）Sc(s) 与 Sc(g)，（b）NH₃(g) 与 NH₃(aq)，（c）1 mol P₄(g) 与 2 mol P₂(g)，（d）C（石墨）与 C（金刚石）。

**解：**

**(a)**

**(b)**

**(c)**

**(d)**
:::


::: {.callout-tip title="习题 19.53"}
利用附录 C 中的 $S^{\circ}$ 值，计算下列反应的 $\Delta S^{\circ}$ 值。每种情况下解释 $\Delta S^{\circ}$ 符号的原因。

(a) $\mathrm{C_{2}H_{4}(g) + H_{2}(g)} \longrightarrow \mathrm{C_{2}H_{6}(g)}$

(b) $\mathrm{N_{2}O_{4}(g)} \longrightarrow 2\mathrm{NO_{2}(g)}$

(c) $\mathrm{Be(OH)_{2}(s)} \longrightarrow \mathrm{BeO(s) + H_{2}O(g)}$

(d) $2\mathrm{CH_{3}OH}(g) + 3\mathrm{O_{2}(g)} \longrightarrow 2\mathrm{CO_{2}(g)} + 4\mathrm{H_{2}O(g)}$

**解：**

**(a)**

**(b)**

**(c)**

**(d)**
:::


## 第 5 节 吉布斯自由能


::: {.callout-tip title="习题 19.8"}
附图显示了某假想反应的 $\Delta G$ 随温度如何变化。（a）在什么温度下系统处于平衡？（b）在哪些温度范围内反应是自发的？（c）$\Delta H$ 为正还是为负？（d）$\Delta S$ 为正还是为负？[第 19.5 节和第 19.6 节]
![](./assets/images/5e2dc6cc9e199162ef75324e18ca23597efd065acd2632f2d513e8e59bafae2a.jpg){fig-align="center"}

温度

**解：**

**(a)**

**(b)**

**(c)**

**(d)**

:::


::: {.callout-tip title="习题 19.57"}
某化学反应的 $\Delta H^{\circ} = -35.4$ kJ，$\Delta S^{\circ} = -85.5$ J/K。（a）该反应是放热还是吸热？（b）该反应使系统的无序度增加还是减小？（c）计算 298 K 时该反应的 $\Delta G^{\circ}$。（d）在 298 K 标准条件下该反应是否自发？

**解：**

**(a)**

**(b)**

**(c)**

**(d)**
:::


::: {.callout-tip title="习题 19.65"}
将下列各反应归类为表 19.3 总结的四种类型之一：（i）所有温度下均自发；（ii）任何温度下均不自发；（iii）低温自发、高温不自发；（iv）高温自发、低温不自发。

(a) $\mathrm{N}_{2}(g) + 3\mathrm{H}_{2}(g) \longrightarrow 2\mathrm{NH}_{3}(g)$; $\Delta H^{\circ} = -91.8\ \mathrm{kJ}$; $\Delta S^{\circ} = -197\ \mathrm{J/K}$

(b) $\mathrm{S}_{8}(s) + 12\mathrm{O}_{2}(g) \longrightarrow 8\mathrm{SO}_{3}(g)$; $\Delta H^{\circ} = -3168\ \mathrm{kJ}$; $\Delta S^{\circ} = 2053.6\ \mathrm{J/K}$

(c) $2\mathrm{POCl}_{3}(g) \longrightarrow 2\mathrm{PCl}_{3}(g) + \mathrm{O}_{2}(g)$; $\Delta H^{\circ} = +572\ \mathrm{kJ}$; $\Delta S^{\circ} = +179\ \mathrm{J/K}$

**解：**

**(a)**

**(b)**

**(c)**
:::


## 第 6 节 自由能与温度


::: {.callout-tip title="习题 19.69"}
某反应的 $\Delta H = -32$ kJ，$\Delta S = -98$ J/K。假设 $\Delta H$ 和 $\Delta S$ 不随温度变化。（a）在什么温度下该反应的 $\Delta G = 0$？（b）若温度高于（a）中的值，反应是自发的还是非自发的？

**解：**

**(a)**

**(b)**
:::


## 第 7 节 自由能与平衡常数


::: {.callout-tip title="习题 19.9"}
考虑反应 $\mathrm{A}_{2}(g) + \mathrm{B}_{2}(g) \rightleftharpoons 2\mathrm{AB}(g)$，图中 A 原子用红色表示，B 原子用蓝色表示。（a）若 $K_{c} = 1$，哪个方框代表平衡时的系统？（b）若 $K_{c} = 1$，哪个方框代表 $Q < K_{c}$ 时的系统？（c）按反应 $\Delta G$ 的大小递增排列这些方框。[第 19.5 节和第 19.7 节]
![](./assets/images/807399bf1e1012fbcc660801eed1029411d972477881aa4d9a843eb2d6e45240.jpg){fig-align="center"}

(1)

![](./assets/images/876947b3776cf9cc3d574b1bced4a1eaaebcfb1b58646ed4d4ebd04d04df1ab1.jpg){fig-align="center"}

(2)

![](./assets/images/edc86bf2a89982e74e2492926859cf746ead5af9a8bd641619cb464c7a7bc570.jpg){fig-align="center"}

(3)

**解：**

**(a)**

**(b)**

**(c)**

:::


::: {.callout-tip title="习题 19.77"}
当 $O_{2}$ 分压减小时，指出下列各反应中 $\Delta G$ 增加、减小还是不变：

(a) $2\mathrm{CO}(g) + \mathrm{O}_{2}(g) \longrightarrow 2\mathrm{CO}_{2}(g)$

(b) $2\mathrm{H}_{2}\mathrm{O}_{2}(l)\longrightarrow2\mathrm{H}_{2}\mathrm{O}(l)+\mathrm{O}_{2}(g)$

(c) $2\mathrm{KClO}_{3}(s)\longrightarrow\mathrm{KCl}(s)+3\mathrm{O}_{2}(g)$

**解：**

**(a)**

**(b)**

**(c)**
:::


::: {.callout-tip title="习题 19.81"}
利用附录 C 中的数据，计算 298 K 时下列各反应的平衡常数 $K$ 和 $\Delta G^{\circ}$：

(a) $\mathrm{H}_{2}(g) + \mathrm{I}_{2}(g) \rightleftharpoons 2 \mathrm{HI}(g)$

(b) $\mathrm{C_{2}H_{5}OH(g)} \rightleftharpoons \mathrm{C_{2}H_{4}(g)} + \mathrm{H_{2}O(g)}$

(c) $3\mathrm{C}_{2}\mathrm{H}_{2}(g) \rightleftharpoons \mathrm{C}_{6}\mathrm{H}_{6}(g)$

**解：**

**(a)**

**(b)**

**(c)**
:::


::: {.callout-tip title="习题 19.83"}
考虑碳酸钡的分解：

$$
\mathrm{BaCO} _{3} (s) \rightleftharpoons \mathrm{BaO} (s) + \mathrm{CO} _{2} (g)
$$

利用附录 C 中的数据，计算（a）298 K 和（b）1100 K 时 $CO_{2}$ 的平衡分压。

**解：**

**(a)**

**(b)**
:::
